src/server/server.js

```diff
@@ -3,21 +3,61 @@
 const getPageTemplate = require("./getPageTemplate");
 
 const http = require("http");
+const path = require("path");
+const fs = require("fs");
+
+const config = require("@carto/routes");
```

Comment:
Note where this comes from?

Reply:
Also, does this only apply in the case where the server is unbundled? This makes the assumption we aren't running the server through a bundle I presume.

Reply:
The server always needs the route manifest, so this is true for both dev server and docker image. Will add comment to note the source